android/java/AndroidManifest.xml

```diff
@@ -155,15 +155,12 @@
     android:noHistory="true"
     android:launchMode="singleTask"/>
 
-<meta-data
```

Comment:
This one is already defined in `billingclient's` manifest.

Reply:
![Screenshot from 2024-07-17 19-04-04](https://github.com/user-attachments/assets/52cac08f-d772-4147-9645-ec0a57e2d7e7)